packages/react-components/src/components/VideoEffects/VideoEffectsItem.tsx

```diff
@@ -128,7 +128,7 @@ export const _VideoEffectsItem = (props: _VideoEffectsItemProps): JSX.Element =>
   const backgroundImage = props.backgroundProps?.url;
   const iconContainerStyles = mergeStyles({ height: '1.25rem' }, props.styles?.iconContainer);
   const textContainerStyles = mergeStyles(
-    { height: '1.25rem', width: '100%', overflow: 'hidden' },
+    { height: '1.25rem', maxWidth: '4.5rem', overflow: 'hidden' },
```

Comment:
I would be curious to know if we a max width constant we can use that's related to the tile itself